Tests/MongoSwiftTests/CommandMonitoringTests.swift

```diff
@@ -10,6 +10,11 @@ final class CommandMonitoringTests: MongoSwiftTestCase {
     }
 
     func testCommandMonitoring() throws {
+        if MongoSwiftTestCase.topologyType == .sharded {
+            print("Skipping test case because of unsupported topology type \(MongoSwiftTestCase.topologyType)")
```

Comment:
Might want to abstract this message into a TestUtils function since we print a similar one in another place.

Reply:
I'm going to punt on this for now, I think we should have a _far_ more explicit, DRY way of determining if a test should be skipped in the future. Let's leave this to that project.